analyses/cell-type-nonETP-ALL-03/.gitignore

```diff
@@ -2,9 +2,8 @@
 /results/*
 !/results/README.md
 
-#Plots should not be committed
-/plots/*
-!/plots/.gitkeep
+# multipanel plots are not committed
+/plots/multipanels_*
```

Comment:
These + the dot plot are the only ones I would commit. I've suggested a change to the annotation script to remove plotting of individual UMAPs

Reply:
Just to clarify, do you mean to upload the multipanels_* to the repository? Even if I change the dpi = 150, the size of these mutipanels_*png ranges from 192kb to 676kb (only one file is below 200kb). Would it be fine if I save all of these multipanels plot as JPEG?

Reply:
Ah, good point! Yes, let's try JPEGs.

Reply:
Even with JPEGs (dpi = 150), the range is from 320kb to 652kb

Reply:
Do you have any other ideas on how to make the size lower than 200kb?

Reply:
We just merged in https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/792, so if you merge `AlexsLemonade:main` into this branch, you should be able to commit the larger PNGs. Please let us know if that doesn't work as expected!

Reply:
I have merged it, but the `multipanels_` png files do not show up in the right panel of GitKraken Desktop (the changes section). So I cannot commit them.

Reply:
If you remove this line from the `.gitignore`, do they show up in GitKraken?

Reply:
Yes, it works! Thank you so much.